napps/kytos/topology/README.rst

```diff
@@ -0,0 +1,350 @@
+Overview
+========
+
+This NApp is responsible for keeping and returning the network topology in a
+protocol-independent way.The topology is built based on the datapaths and hosts
+known to the controller.
+Topology is exported in JSON format via REST endpoints.
+
+
+Requirements
+============
+
+All requirements are listed in *requirements.txt*.
+
+
+Installing
+==========
+
+All of the Kytos Network Applications are located in the NApps online repository.
+To install this NApp, run:
+
+.. code:: shell
+
+   $ kytos napps install kytos/topology
+
+
+Settings.py paths
+=================
+
+Here is an example of how paths may be defined by the user using the ``paths``
+dictionary in the settings.py file of the **topology** NApp. The path alias is
+defined as a key, and the links of the path and the customized metrics are
+defined in the associated value.
+
+Links are defined as "<datapath>:<port>-<datapath>:<port>", where the <datapath>
+value is the DPID without ":" and without left-side zeroes. Aliases of other
+links can also be used to represent link compositions.
+
+.. code-block:: python3
+
+  paths = {
+           "1-to-2":{
+               "links": ["1:2-2:1"],
+               "metrics": {
+                   "custom_cost": 8
+               }
+            }
+           "2-to-1":{
+               "links": ["2:1-1:2"],
+               "metrics": {
+                   "custom_cost": 8
+                   "reliability": 0.98
+               }
+            }
+           "1-to-3":{
+               "links": ["1to2","2:3-3:1"],
+               "metrics": {
+                   "custom_cost": 15
+               }
+            }
+          }
+
+
+REST API
+========
+
```

Comment:
Now, all our NApps has the .yml file where the documentation will be displayed on our napps repository.

So all README files with REST API section should point to the NApp repository link.